chain/beacon/beacon.go

```diff
@@ -43,20 +43,21 @@ type BeaconPoint struct {
 // been posted on chain.
 type RandomBeacon interface {
 	Entry(context.Context, uint64) <-chan Response
-	VerifyEntry(types.BeaconEntry, types.BeaconEntry) error
+	VerifyEntry(entry types.BeaconEntry, prevEntrySig []byte) error
 	MaxBeaconRoundForEpoch(network.Version, abi.ChainEpoch) uint64
 }
 
 func ValidateBlockValues(bSchedule Schedule, nv network.Version, h *types.BlockHeader, parentEpoch abi.ChainEpoch,
 	prevEntry types.BeaconEntry) error {
-	{
+	// Before nv22 we had "chained" beacons, and so required two entries at a fork
+	if nv < network.Version22 {
```

Comment:
If drand network upgrade does not align with chain network upgrade, should we check the exact drand network upgrade epoch here?

Reply:
This is a fork logic which works off the DrandSchedule. So, the meaning of this is to disable the old fork logic before switching to the new Drand network.